### DISCUSSION TITLE
How do you change the default Google font?

### DISCUSSION THREAD

**Discussion by varna9000** [getnikola/nikola#3526] [Q&A] [answered]
My project is based on bootstrap4 theme. I tried to change the font from ``book.tmpl``, which is located in ``templates`` folder of my project:

```
<link href="https://fonts.googleapis.com/css2?family=Lobster&display=swap" rel="stylesheet"> 
```

 then I initialised the font in files/assets/css/custom.css like this:

```
body {
  font-family: 'Lobster', cursive !important;
}
```

But the font is not changed after rebuilding. I can't see the font loading in the ``<head>`` of the source after compilation either. Is there a specific way to load a new google font?

**Comment by Kwpolska** [comment, marked as answer]
1. Check if the `<link>` is visible in your site source
2. Check if the CSS change is visible (in `all.css`/`all-nocdn.css`)
3. Force refresh (Ctrl-Shift-R)

PS. For usage questions, we prefer the mailing list (https://groups.google.com/group/nikola-discuss) or the Discussions tab (which I enabled recently as an experiment). I migrated this to Discussions.
PPS. Consider not using Lobster as your body font? 😅

**Reply by varna9000** [reply]
Yeah, I know, I wouldn't have Lobster in production :) 
I'm using it so I can debug and distinguish the changes. I'll check the other suggestion and revert.

**Comment by varna9000** [comment]
Ok, so the ``<link>`` is not being generated in the source. The  additional **body** style appears in all-nocdn.css.
If I put the ``<link>`` in base.tmpl, just before ``</head>``, then the new font is loaded. Why book.tmpl is being ignored on build?

**Reply by Kwpolska** [reply]
`book.tmpl` applies only to the book example page and not to all pages. If you want this to appear everywhere, modify `base.tmpl` (`nikola theme -c base.tmpl` can help) or put it in `EXTRA_HEAD` in `conf.py`.

**Reply by varna9000** [reply]
Thanks! I appreciate your help. I'm sure I'll have more question and I'll ask them here. Nikola is great!
